README.md

```diff
@@ -135,11 +133,151 @@ Here's an example of importing timeseries for one day only:
 
 Please see more about time filtering [here](https://docs.influxdata.com/influxdb/v1.7/query_language/schema_exploration#filter-meta-queries-by-time).
 
-#### Performance
+
+## Migrating data from Prometheus
```

Comment:
misleading title (prometheus) and help command below (influx)

Reply:
Hm, I don't see which command below you're pointing to. Could you pls elaborate?

Reply:
`./vmctl influx --help` line 140

Reply:
yes, now I see it) thx)